HorseRace/HorseRace/ResultView.swift

```diff
@@ -284,6 +295,32 @@ struct ResultView: View {
                         }
                     }
                 }
+            } else if num == 7 {
+                HStack(alignment: .top, spacing: 20) {
+                    VStack(spacing: 8) {
+                        ForEach(0..<4, id: \.self) { i in
+                            content(i, rankingInfo[i].horseNum, rankingInfo[i].second)
+                        }
+                    }
+                    VStack(spacing: 8) {
+                        ForEach(4..<7, id: \.self) { i in
+                            content(i, rankingInfo[i].horseNum, rankingInfo[i].second)
+                        }
+                    }
```

Comment:
```suggestion
                    VStack(spacing: 8) {
                        ForEach(4..<7, id: \.self) { i in
                            content(i, rankingInfo[i].horseNum, rankingInfo[i].second)
                        }
                        content(0, rankingInfo[0].horseNum, rankingInfo[0].second)
                            .opacity(0)
                    }
```
이런식으로 수정하면 5명인 화면과 통일감있게 보일 것 같네요!
![image](https://user-images.githubusercontent.com/45297745/208642304-f015e1b1-3c15-45a5-b379-f3b117ffcf48.png)